pkg/controllers/scheduler/framework/plugins/apiresources/apiresources.go

```diff
@@ -39,5 +39,5 @@ func (pl *APIResources) Filter(ctx context.Context, su *framework.SchedulingUnit
 			return framework.NewResult(framework.Success)
 		}
 	}
-	return framework.NewResult(framework.Unschedulable, "No matched group version kind.")
+	return framework.NewResult(framework.Unschedulable, "cluster(s) didn't have the matched group version kind")
```

Comment:
```suggestion
	return framework.NewResult(framework.Unschedulable, "cluster(s) didn't support this APIVersion")
```
Would this message be clearer to users?

Reply:
Make senses.